user/views.py

```diff
@@ -186,3 +186,32 @@ def get(self,request):
 
         except User.DoesNotExist:
             return JsonResponse({'message':'USER DOES NOT EXIST'}, status=400)
+        except ValueError:
+            return JsonResponse({'message': 'VALUE_ERROR'}, status=400)
+
+class KakaoLoginView(View):
+    def get(self,request):
+        access_token = request.headers['Authorization']
+        kakao_header = {'Authorization':f'Bearer {access_token}'}
+
+        url          = 'https://kapi.kakao.com/v2/user/me'
+        response     = requests.get(url, headers=kakao_header)
+        user         = response.json()
+
+        if user['kakao_account']['profile'].get('profile_image_url'):
+            image = user['kakao_account']['profile']['profile_image_url']
+        else:
+            image = None
+
+        if user.get('id'):
+            user = User.objects.get_or_create(
+                social_login_id = user.get('id'),
+                name            = user['kakao_account']['profile']['nickname'],
+                social          = SocialPlatform.objects.get(platform='kakao'),
+                image_url       = image
+                )[0]
+            access_token = jwt.encode({'id': user.id},SECRET_KEY,algorithm= ALGORITHM).decode('utf-8')
+
+            return JsonResponse({"Authorization": access_token}, status=200)
```

Comment:
응답으로 주는 token이 어떤 token인지 정확히 알려주는게 좋을 것 같습니다.
나중에 refresh_token.. 등 다른 인증절차가 추가될 경우 어떤 token인지 정확히 명시해주는 것이 좋습니다!
```suggestion
            return JsonResponse({"access_token": access_token}, status=200)
```

Reply:
넵 수정했습니다!